### PR TITLE
Jakartaee-9: Upgrade app client features

### PR DESCRIPTION
**Pull request by cbridgha**
This is a partial delivery of the App Client activation - still waiting on a few dependencies, and have disabled the repeat tests for those scenarios.

Also needed to adjust the visibility of a few of the features which were full, but should be base or core.

### CODE REVIEW

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
#build

**Comment by cbridgha** [comment]
This build finished clean with one unrelated issue:  https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_1mNswc27EeqmIcsdsdUKbw&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

File: dev/com.ibm.ws.clientcontainer.8.0_fat/build.gradle [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+addRequiredLibraries.dependsOn addJakartaTransformer
```
Comment (kwsutter) [inline dev/com.ibm.ws.clientcontainer.8.0_fat/build.gradle:12]: Is this okay that there's no EOL here?

**Comment by cbridgha** [comment]
#libby

File: dev/com.ibm.ws.clientcontainer.8.0_fat/build.gradle (hunk 1 above)
Comment (cbridgha) [reply dev/com.ibm.ws.clientcontainer.8.0_fat/build.gradle:12]: yeah think so - its def working because the fat test wouldn't work without it

**Pull request merged**